src/ctransaction.cpp

```diff
@@ -123,6 +123,7 @@ void CTransaction::addData(const char * data) {
         case 110:
         case 115:
             newParameter.type = TYPE_STRING;
+            break;
```

Comment:
Can you confirm this break is correct? It looked like it to me.

Reply:
This should be correct, as it is for the Chat Subject parameter.